keyboards/keyboardio/atreus/keymaps/slf-atreus/keymap.c

```diff
@@ -0,0 +1,266 @@
+		     KC_F24,       LALT(KC_F3), KC_NO,      KC_NO,    TD(L_BRAX),                        TD(R_BRAX),  KC_LEFT,   KC_DOWN,  KC_UP,   KC_RGHT,
+		     LCTL(KC_F12), LALT(KC_X),  KC_NO,      KC_NO,    KC_NO,      TD(QUOTS),  TD(TACHE), KC_MINS,     KC_EQL,    KC_COMM,  KC_DOT,  KC_ENT,
+		     KC_TRNS,      KC_NO,       KC_NO,      MO(3),    KC_NO,      KC_NO,      KC_NO,     KC_NO,       KC_NO,     KC_NO,    KC_NO,   LSFT(KC_TAB)),
+	
+	[2] = LAYOUT(KC_1,         KC_2,        KC_3,       KC_4,     KC_5,                              KC_6,        KC_7,      KC_8,     KC_9,    KC_0,
+		     KC_NO,        LALT(KC_F3), KC_NO,      KC_NO,    TD(L_BRAX),                        TD(R_BRAX),  KC_HOME,   KC_PGDN,  KC_PGUP, KC_END,
+     		     LCTL(KC_F12), LALT(KC_X),  KC_NO,      KC_NO,    KC_NO,      TD(QUOTS),  TD(TACHE), KC_MINS,     KC_EQL,    KC_COMM,  KC_DOT,  KC_NO,
+		     KC_TRNS,      KC_NO,       KC_NO,      KC_NO,    MO(3),      KC_NO,      KC_NO,     KC_NO,       MO(3),     KC_NO,    KC_NO,   KC_TRNS),
+	
+	[3] = LAYOUT(KC_F1,        KC_F2,       KC_F3,      KC_F4,    KC_F5,                             KC_F6,       KC_F7,     KC_F8,    KC_F9,   KC_F10,
+		     AS_TOGG,      KC_F24,      KC_NO,      KC_NO,    KC_NO,                             KC_NO,       LSA(KC_LEFT), LSA(KC_DOWN), LSA(KC_UP), LSA(KC_RGHT),
+		     KC_LCTL,      LALT(KC_DEL),KC_NO,      KC_NO,    KC_NO,      KC_NO,      KC_VOLU,   KC_NO,       KC_F12,    KC_F13,   KC_F14,  KC_NO,
+		     KC_TRNS,      KC_F23,      KC_LCTL,    KC_NO,    KC_TRNS,    KC_NO,     KC_VOLD,    KC_MPLY,     KC_LSFT,   KC_NO,   KC_NO,  KC_TRNS)
+};
+
+
+// Two combos
+const uint16_t PROGMEM CtlAlt_combo_L[] = {LCTL_T(KC_ESC), KC_RCTL, COMBO_END};
+const uint16_t PROGMEM CtlAlt_combo_R[] = {KC_RCTL, LCTL_T(KC_TAB), COMBO_END};
+combo_t key_combos[COMBO_COUNT] = {
+    COMBO(CtlAlt_combo_L, KC_LALT), // Alt using tab/Ctl and key to left
+    COMBO(CtlAlt_combo_R, KC_LALT), // Alt using Esc/Ctl and key to right
+};
+
+
+/* 
+Notes on keys
+or, why on earth do I have keymap 
+lower-shift-z is CTRL-F12 - Darkroom mode in emacs
+lower-x is ALT-X - emacs
+layer3-a is toggle autoshift
+layer3-s is F24 - i3 controls popup reminder
+lower-s is ALT-F3 - multi-select in Sublime text
+*/
+
+
+// Determine the tapdance state to return
+uint8_t cur_dance(tap_dance_state_t *state) {
+    if (state->count == 1) {
+        return SINGLE_TAP;
+    } else if (state->count == 2) {
+        return DOUBLE_TAP;
+    } else if (state->count == 3) {
+        return TRIPLE_TAP;
+    } else {
+        reset_tap_dance (state);
+        return 3;
+    }
+}
+
+
+// Handle the possible states for each tapdance keycode you define:
+
+void lbrax_finished(tap_dance_state_t *state, void *user_data) {
+    td_state = cur_dance(state);
+    switch (td_state) {
+        case SINGLE_TAP:
+            register_mods(MOD_BIT(KC_LSFT)); // For a layer-tap key, use `layer_on(_MY_LAYER)` here
+            register_code16(KC_9);
```

Comment:
You can do this, instead: 
```suggestion
            register_code16(LSFT(KC_9));
```